torch/_dynamo/variables/tensor.py

```diff
@@ -199,6 +165,40 @@ def var_getattr(self, tx, name):
         if result is not None and self.source is not None:
             result = result.add_guard(self.make_guard(GuardBuilder.TYPE_MATCH))
 
+        # For attributes (not methods) that were not caught in the special handling above,
+        # (e.g. tensor.real), we handle these generically, assuming that the output type is
+        # a tensor.
+        if result is None:
+
+            def try_generic_attr_handling():
+                from .builder import wrap_fx_proxy
+                from .misc import GetAttrVariable
+
+                try:
+                    static_attr = inspect.getattr_static(torch.Tensor, name)
+                except NameError:
+                    return None
+
+                # Make sure this is an attribute, not a method.
+                # type(torch.Tensor.H) should be "getset_descriptor"
+                # This is a because of CPython implementation, see THPVariableType:
+                # these attributes are implemented under tp_getset, which appear
+                # as `getset_descriptor`s, (compared to, say, methods which appear
+                # as `method_descriptor`s)
+                if not (
+                    str(getattr(type(static_attr), "__name__", None))
+                    == "getset_descriptor"
```

Comment:
ugh, does it have to be a string comparison here?

Reply:
I'd also be OK if we just have a hardcoded list of Tensor properties here too

Reply:
found it at `types.GetSetDescriptorType`